src/prisma/schema.prisma

```diff
@@ -1939,3 +1941,49 @@ model AdditionalBlockQuota {
   blockCount Int
   createdAt  DateTime  @default(now())
 }
+
+model Form {
```

Comment:
general note / question - why dont we prefix form related models with Proposal? They will not be usable anywhere else as they hold proposal relation anyway. 
i.e. ProposalForm, ProposalFormFieldType, ProposalFormField

Reply:
We decided to not append Proposal since its generic atm.